### ISSUE TITLE
Depends on "future"

### ISSUE THREAD

**Issue by eliotb** [iogf/untwisted#6]
pip installed untwisted  as dependency of "vy"

> Error:
>   File "/home/eliot/.local/lib/python3.5/site-packages/untwisted/core.py", line 6, in <module>
>     from future.utils import itervalues
> ImportError: No module named 'future'

`pip install future` fixed this

So, untwisted should depend on 'future' ?

**Comment by iogf** [comment]
Yes, it works for both python2 and python3, i'm planning to get rid of python2 support soon, for now, it is necessary to have future package installed.

**Comment by iogf** [comment]
I haven't added that in the requeriments.txt file i'll be adding that soon.
